src/assets/moves.ts

```diff
@@ -4109,7 +4118,8 @@ export default new GraphQLCollection<string, Pokemon.Move>([
       target: 'Any',
       type: 'Flying',
       contestType: 'Clever',
-      maxMovePower: 130
+      maxMovePower: 130,
+      isFieldMove: 'The Pokemon can fly the player to any previously visited town/landmark. Fly can be taught to Pokemon with HM02 in all generations.'
```

Comment:
"Pokemon" -> "Pokémon"

Also check if the same typo is elsewhere in items & abilities please. I'm pretty sure I did the rest a long while ago.